java-checks/src/main/java/org/sonar/java/checks/AnonymousClassShouldBeLambdaCheck.java

```diff
@@ -76,14 +78,44 @@ public void visitNewClass(NewClassTree tree) {
   }
 
   private static boolean isSAM(ClassTree classBody) {
-    if(hasOnlyOneMethod(classBody.members()) && classBody.symbol().isTypeSymbol()) {
+    if (hasOnlyOneMethod(classBody.members()) && classBody.symbol().isTypeSymbol()) {
       // Verify class body is a subtype of an interface
+      MethodJavaSymbol methodSymbol = ((MethodJavaSymbol) ((MethodTree) classBody.symbol().memberSymbols().stream().findFirst().get().declaration())
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1946&issues=AWI_D3ZNgCR0FH6J7UAL&open=AWI_D3ZNgCR0FH6J7UAL)

[ISSUE_KEY]: AWI_D3ZNgCR0FH6J7UAL